python/tests/conftest.py

```diff
@@ -112,7 +113,7 @@ def lmdb_library(lmdb_storage, lib_name):
 
 @pytest.fixture(scope="session")
 def s3_storage_factory():
-    with MotoS3StorageFixtureFactory(use_ssl=S3_SSL_TEST_ENABLED) as f:
+    with MotoS3StorageFixtureFactory(use_ssl=SSL_TEST_ENABLED) as f:
```

Comment:
It's too confusing that `SSL_TEST_ENABLED` only applies to S3 and not to Azure.

Reply:
Add docstring to explain